ctru-rs/src/services/cam.rs

```diff
@@ -0,0 +1,1026 @@
+use bitflags::bitflags;
+use ctru_sys::Handle;
+
+pub struct Cam(());
+
+bitflags! {
+    #[derive(Default)]
+    pub struct CamPort: u32 {
+        const PORT_NONE = 0;
```

Comment:
You should use `ctru-sys` constants rather than hardcoded numbers.
This goes for all enums in the new module.